config/envoyconfig/luascripts/set-client-certificate-metadata.lua

```diff
@@ -0,0 +1,14 @@
+function envoy_on_request(request_handle)
+    local metadata = request_handle:streamInfo():dynamicMetadata()
+    local ssl = request_handle:streamInfo():downstreamSslConnection()
+    metadata:set("com.pomerium.client-certificate-info", "presented",
```

Comment:
Let me know if you have any opinions on the metadata namespace to use for this data. The only guidance I've found from the Envoy documentation is this:

> Key is the reverse DNS filter name, e.g. com.acme.widget. The envoy.* namespace is reserved for Envoy’s built-in filters.

(from https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/base.proto#envoy-v3-api-field-config-core-v3-metadata-filter-metadata)

However, this metadata is for consumption by our ExtAuthz service, which can apparently ask for [any metadata namespaces it likes](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ext_authz/v3/ext_authz.proto.html#envoy-v3-api-field-extensions-filters-http-ext-authz-v3-extauthz-metadata-context-namespaces), so I don't think this needs to correspond exactly to a filter name.

Again, happy to change this if you have an opinion about what it should be.